images/Makefile

```diff
@@ -35,8 +81,9 @@ build-all:
 	$(MAKE) ${OP} WHAT=alpine
 	$(MAKE) ${OP} WHAT=opensuse      RELEASE=leap       IS_LATEST=true
 	$(MAKE) ${OP} WHAT=opensuse      RELEASE=tumbleweed
-	$(MAKE) ${OP} WHAT=ubuntu        RELEASE=16.04
-	$(MAKE) ${OP} WHAT=ubuntu        RELEASE=18.04      IS_LATEST=true
-	$(MAKE) ${OP} WHAT=ubuntu        RELEASE=19.04
+	$(MAKE) ${OP} WHAT=ubuntu        RELEASE=16.04						IS_MANIFEST_LIST=0
+	$(MAKE) ${OP} WHAT=ubuntu        RELEASE=18.04      IS_LATEST=true	IS_MANIFEST_LIST=1 GOARCH=arm64
+	$(MAKE) ${OP} WHAT=ubuntu        RELEASE=18.04      IS_LATEST=true	IS_MANIFEST_LIST=1 GOARCH=amd64
```

Comment:
Ack -- only adding am arm64/aarch64 userspace image for Ubuntu 18.04 starting out